ivy/functional/frontends/numpy/linalg/matrix_and_vector_products.py

```diff
@@ -57,6 +57,12 @@ def einsum(
     return ivy.einsum(subscripts, *operands, out=out)
 
 
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def gradient(a, *, axis=None, edge_order=1):
```

Comment:
Could you add the `varargs` argument that is present in the[ `np.gradient` function](https://numpy.org/doc/stable/reference/generated/numpy.gradient.html). Also, let's rename `a` to `f` since in numpy it's the latter :)